023-AdvancedKubernetes/Coach/01-setup.md

```diff
@@ -1,6 +1,6 @@
-# Challenge 1 - Setup
+# Challenge 1: Coach's Guide - Setup
 
-[Next Challenge>](./02-helm.md)
+[Next Challenge>](./02-helm.md) - **[Home](README.md)**
```

Comment:
Link to next challenge is correct, but in wrong location.  Move it to the right side of the "home" link.

Reply:
Fixed